### PR TITLE
[contract-manager] General improvements

### PR DESCRIPTION
**Pull request by m30m**

### CODE REVIEW

File: contract_manager/src/base.ts [hunk 1]
```diff
@@ -1,6 +1,22 @@
 import { DataSource } from "xc_admin_common";
 import { Chain } from "./chains";
 
+export interface TxResult {
+  id: string;
+  info: any; // chain specific info
+}
+
+export type DeploymentType = "stable" | "edge";
```
Comment (ali-bahjati) [inline contract_manager/src/base.ts:9]: let's rename "edge" to "beta".

File: contract_manager/src/contracts/aptos.ts [hunk 2]
```diff
@@ -44,23 +44,28 @@ export class AptosContract extends Contract {
     return this.sendTransaction(senderPrivateKey, txPayload);
   }
 
-  private sendTransaction(
-    senderPrivateKey: string,
+  private async sendTransaction(
+    senderPrivateKey: PrivateKey,
     txPayload: TxnBuilderTypes.TransactionPayloadEntryFunction
   ) {
```
Comment (ali-bahjati) [inline contract_manager/src/contracts/aptos.ts:50]: I know it's inferable but i think specifying the type is more clear.

**Review by ali-bahjati — APPROVED** [review]

File: contract_manager/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
```
Comment (ali-bahjati) [inline contract_manager/.eslintrc.js:7]: Can't we instead turn it off for the only place that it's defined? In general it's good to keep this enabled by default?

Comment (m30m) [reply contract_manager/.eslintrc.js]: We had ~40 warnings, fixed most and ignored 4 of them

**Review by ali-bahjati — APPROVED** [review]

**Pull request merged**